### DISCUSSION TITLE
Add [spoiler] tags to formatted text input areas

### DISCUSSION THREAD

**Discussion by luchaos** [RetroAchievements/RAWeb#725] [Ideas]
Posting solutions like on http://retroachievements.org/Game/8617 should be either put under a spoiler tag by the user (with a proper description), or have a spoiler tag applied by mods. Currently they have to be removed to not accidently ruin the experience for users that want to figure out things themselves.

This would be in place for any text area that allows formatted text input on the site (comments, forum posts, etc)

Required in preparation for an extensive future guides section as well.

**Comment by Ben-Ghazi** [comment]
I can see this being useful if it can apply to achievement descriptions, particuarly with story-heavy games but I don't see this helping with the issue that prompted the discussion.

**Comment by Jamiras** [comment]
We have [spoiler] tags for forums. #756 is the request for spoilers in achievement descriptions. It's probably still desirable to have spoilers in comments.
